scripts/app.js

```diff
@@ -31,7 +34,7 @@
       $.getJSON('/data/projects.json', function(rawData){
         Project.loadAll(rawData);
         localStorage.setItem('rawData', JSON.stringify(rawData));
-        next();
+        ProjectView.initIndexPage();
       });
     }
   };
```

Comment:
refactor comment: removing .forEach/.push methods and changing it to .map so that my data will be pushed into an array and will be displayed dynamically. the .forEach and .push methods seems to add another step to the .toHtml process.